make/stratos/run

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+. "${BASH_SOURCE%/*}/_env"
+. "${BASH_SOURCE%/*}/_has_release.sh"
+
+# TODO(f0rmiga): Refactor make/run and this file to move the following variables to a single source.
+: "${DOMAIN:=cf-dev.io}"
+: "${UAA_HOST:=uaa.${DOMAIN}}"
+: "${UAA_PORT:=2793}"
+: "${UAA_CLIENT_ID:=cf}"
+: "${UAA_CLIENT_SECRET:=""}"
+: "${UAA_ADMIN_IDENTIFIER:=cloud_controller.admin}"
+: "${UAA_SKIP_SSL_VALIDATION:=true}"
+: "${STORAGE_CLASS:=$(kubectl get storageclass 2>/dev/null | awk '/(default)/ { print $1 ; exit }')}"
+
+tmp_stratos="/tmp/stratos"
+mkdir -p $tmp_stratos
```

Comment:
Hmm, having a hard-coded temp directory (as opposed to `mktemp`) smells bad.

Reply:
I had a `mktemp` all the development cicle, decided to move to a hardcoded one just before commit. Any particular reason why it is bad?

Reply:
Nah, it was just a gut check thing; if you've thought about it and have specifically decided this was better, then it's fine.

Reply:
Just makes it slightly easier to do a caching later when we see need.